.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Test and Build React
+
+on:
+    push:
+
+    pull_request:
+        branches: ['main']
+
+    workflow_dispatch:
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v3
```

Comment:
Just to make the process easier to understand for those newer to development, I wonder if you should add a name to each of the steps to help debug problems.

I don't forsee there being any problems with the generic checkout/setup node actions, but it might make it easier for people to understand what's going on.

Thoughts here?

Reply:
Sure why not